sdks/java/io/hadoop-input-format/README.md

```diff
@@ -0,0 +1,167 @@
+<!--
```

Comment:
based on the discussion in https://lists.apache.org/thread.html/808dc653e752705ab585b3593bc76c6c9a167c2d78604b0fdade05d6@%3Cdev.beam.apache.org%3E - please change the directory name hosting the parent files from hadoop-input-format to hadoop

also, I'm not a huge fan of the name hifio for the sub directory. I suspect there may be some discussion of the name, but something like "main" might be better. @iemejia  any thoughts on that?